diff --git EntityBundle.php EntityBundle.php
index f073fdb..e7e9933 100644
--- a/core/lib/Drupal/Core/Entity/Plugin/Condition/EntityBundle.php
+++ a/core/lib/Drupal/Core/Entity/Plugin/Condition/EntityBundle.php
@@ -89,7 +89,9 @@ class EntityBundle extends ConditionPluginBase implements ContainerFactoryPlugin
     }
     /** @var \Drupal\Core\Entity\ContentEntityInterface $entity */
     $entity = $this->getContextValue($this->getDerivativeId());
-    return !empty($this->configuration['bundles'][$entity->bundle()]);
+    if($entity){
+      return !empty($this->configuration['bundles'][$entity->bundle()]);
+    }
   }
 
   /**
